elasticdl/python/common/model_handler.py

```diff
@@ -44,6 +44,15 @@ def _convert_embedding_table_to_numpy_array(embedding_table, embedding_shape):
     return embedding_weights
 
 
+def _is_big_embedding(layer):
```

Comment:
Please make this API compatible with feature column

Reply:
I will implement the replacement for `embedding_column` in the next PR.